src/webgpu/api/validation/capability_checks/limits/maxComputeWorkgroupsPerDimension.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { keysOf } from '../../../../../common/util/data_tables.js';
+
+import { kLimitBaseParams, makeLimitTestGroup } from './limit_utils.js';
+
+const limit = 'maxComputeWorkgroupsPerDimension';
+export const { g, description } = makeLimitTestGroup(limit);
+
+const CreateComputePipelineTypes = {
+  createComputePipeline: true,
+  createComputePipelineAsync: true,
+};
+type CreateComputePipelineType = keyof typeof CreateComputePipelineTypes;
+const kCreateComputePipelineTypes = keysOf(CreateComputePipelineTypes);
+
+async function createComputePipeline(
+  device: GPUDevice,
+  descriptor: GPUComputePipelineDescriptor,
+  pipelineType: CreateComputePipelineType
+) {
+  switch (pipelineType) {
+    case 'createComputePipeline':
+      return device.createComputePipeline(descriptor);
+    case 'createComputePipelineAsync':
+      return await device.createComputePipelineAsync(descriptor);
+  }
+}
+
+g.test('createComputePipeline,async,at_over')
+  .desc(`Test using createComputePipeline,async at and over ${limit} limit`)
```

Comment:
Test name and description should be `dispatchWorkgroups` because that's where the validation is.

BTW it's probably worth making a brief note that dispatchWorkgroupsIndirect is not tested because it's not a validation error if that exceeds the limits.

Reply:
done